src/components/SuccessAnimation/SuccessAnimation.tsx

```diff
@@ -0,0 +1,15 @@
+import { Player } from '@lottiefiles/react-lottie-player';
+import CelebrationCheck from '../../assets/styles/CelebrationCheck.json';
```

Comment:
move `CelebrationCheck` into this file, there's no reason to make it an external file because you're not reusing it.